server/src/controllers/middleware/getTopRatedCourses.js

```diff
@@ -0,0 +1,12 @@
+const { getTopCourses } = require('../../database/queries');
+
+const getTopRatedCourses = async (req, res, next) => {
+  try {
+    const { rows } = await getTopCourses();
+    res.json(rows);
```

Comment:
**handle all cases of rows then send the result** 
* empty array
* array with data